src/D8/MenuLinkContentTrait.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace IntegratedExperts\BehatSteps\D8;
+
+use Behat\Gherkin\Node\TableNode;
+use Drupal\menu_link_content\Entity\MenuLinkContent;
+
+/**
+ * Trait MenuLinkContentTrait.
+ *
+ * @package IntegratedExperts\BehatSteps\D8
+ */
+trait MenuLinkContentTrait {
+
+  /**
+   * Remove menu.
+   *
+   * Provide menu data in the following format:
+   *
+   * | id        |
+   * | 9991      |
+   * | ...       |
+   *
+   * @Given no menu_links:
+   */
+  public function menuLinksDelete(TableNode $table) {
+    foreach ($table->getColumn(0) as $id) {
+      $menu_link = MenuLinkContent::load($id);
+      if ($menu_link) {
+        $menu_link->delete();
+      }
+    }
+  }
+
+  /**
+   * Create a menu if one does not exist.
+   *
+   * Provide menu data in the following format:
+   *
+   * | id    | uuid   | title        | description     | menu_name | enabled |
```

Comment:
please fix the format of this comment. it should be set in a single line to show the table correctly